test/unit/test_pretty_print.py

```diff
@@ -0,0 +1,30 @@
+import re
+import time
+from exasol.ds.sandbox.lib import pretty_print
+from datetime import datetime, timedelta
+
+
+def test_elapsed_rounded():
+    start = datetime.now()
+    time.sleep(1.2)
```

Comment:
Sleeping in a simple unit test feels a waste of time. Could you change the API to avoid this?

Reply:
We should not implement that by our self, then the tests are also gone. See, my previous comments regarding the packages stopwatch and humanfriendly